src/storage/global.js

```diff
@@ -0,0 +1,15 @@
+// documentService.js
+const DEFAULT_TTL_IN_MICROSECONDS = 30 * 24 * 60 * 60 * 1000; // 30 Days
+const MAX_TTL_IN_MICROSECONDS = 90 * 24 * 60 * 60 * 1000; // 90 Days
+
```

Comment:
"global.js" what does this name tell you about what's in here?
ans: nothing

Reply:
ok. i removed the global/helper.js. moved the variables into try-catch.js. check below.